collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -474,6 +474,15 @@ var-log() {
   mkdir -p $TMPDIR/systemlogs
   cp -p /var/log/syslog* /var/log/messages* /var/log/kern* /var/log/docker* /var/log/system-docker* /var/log/cloud-init* /var/log/audit/* $TMPDIR/systemlogs 2>/dev/null
 
+  for STAT_PACKAGE in sysstat atop
```

Comment:
SLES/Oracle/RHEL/CentOS uses location `/va/log/sa` 
can we add `sa` to the loop?